crates/wasmlink/build.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::{Command, Stdio};
+
+fn main() {
+    println!("cargo:rerun-if-changed=runtime/src");
+    println!("cargo:rerun-if-changed=runtime/Cargo.toml");
+    println!("cargo:rerun-if-changed=runtime/Cargo.lock");
+
+    let mut cmd = Command::new("cargo");
+
+    cmd.args(&["build", "--target=wasm32-wasi"]);
```

Comment:
Oh below I thought this may not use WASI, but apparently it does! I think though that to make this module as lean-as-possible we may want it to use the `wasm32-unknown-unknown` target?

Reply:
I'll see about targeting it to `wasm32-unknown-unknown`.  Are there any other tips that would reduce the size of the module any maybe scrape off any unnecessary baggage? My knowledge is a little lacking on how to get the Rust toolchain to spit out the leanest Wasm module possible.

Reply:
The main thing with unknown-unknown is that it's all Rust so it's much easier to use full-crate-graph LTO which enables as many optimizations as possible to produce a small module (basically we give LLVM total knowledge). 

I figured though that we'd want to do this for functionality reasons I mentioned above, but in any case I think it's fine to go either way on this.